### ISSUE TITLE
Implement Search

### ISSUE THREAD

**Issue by bschoenfeld** [Code4HR/hrt-bus-finder#4]
You should be able to search for an intersection and get stops close to it.

**Comment by kmcurry** [comment]
We'll need this to either be completed or the search box removed before we can launch. 

See launch plan in wiki:
https://github.com/c4hrva/hrt-bus-finder/wiki/App-Launch-Marketing-Experiment

**Comment by wbprice** [comment]
Per Ben, this is being given a lower priority in favor of getting the maps working with the new interface.  I'm just going to hide the search module in the meantime (#25) and can add it back in version 1.1.

**Comment by kmcurry** [comment]
:thumbs:

**Comment by kmcurry** [comment]
: thumbsup:

**Comment by kmcurry** [comment]
Waaaa

**Comment by bschoenfeld** [comment]
Sorry for the miscommunication. I want this in the beta release. I think
it's vital to be able to type in a route or a street address and get
information for that location. I was saying that we should have a search
button instead of a search text box. It should pop up a modal that tells
the user to type in an intersection and choose a city from a drop down or
choose a route from a drop down.

**Comment by wbprice** [comment]
Glad to see I'm doing the right thing for the wrong reasons.  Just to clarify, the only thing that's going away in #25 is the search input box.  The search button on the right side is still in place.

**Comment by bschoenfeld** [comment]
#25 actually does hide the entire search, but I merged it anyway. I will
add the search button back in after I implement it.

**Comment by wbprice** [comment]
We might explore putting the search button where the "find current location" button is now and then put the "current location" function in the modal. I would think having both on the same bar would be confusing.

**Comment by bschoenfeld** [comment]
Fixed in #82
